src/plugins/maps_dashboards/public/components/app.tsx

```diff
@@ -34,74 +17,17 @@ interface MapsDashboardsAppDeps {
   navigation: NavigationPublicPluginStart;
 }
 
-export const MapsDashboardsApp = ({
-  basename,
-  notifications,
-  http,
-  navigation,
-}: MapsDashboardsAppDeps) => {
-  const find = async (num: number) => {
-    let hits: never[] = [];
-    await http.post('/api/maps_dashboards/example').then((res) => {
-      hits = res.hits;
-    });
-    return Promise.resolve({
-      total: num,
-      hits: hits,
-    });
-  };
-
-  const tableColumns = [
-    {
-      field: 'title',
-      name: i18n.translate('maps.listing.table.titleColumnName', {
-        defaultMessage: 'Title',
-      }),
-      sortable: true,
-    },
-    {
-      field: 'description',
-      name: i18n.translate('maps.listing.table.descriptionColumnName', {
-        defaultMessage: 'Description',
-      }),
-      sortable: true,
-    },
-  ];
-
+export const MapsDashboardsApp = ({ basename, notifications, http }: MapsDashboardsAppDeps) => {
   // Render the application DOM.
   return (
-    <Router basename={basename}>
+    <Router basename={'/' + basename}>
       <I18nProvider>
-        <>
-          <EuiPage restrictWidth="1000px">
-            <EuiPageBody component="main">
-              <EuiPageContentBody>
-                <TableListView
-                  headingId="mapsListingHeading"
-                  createItem={useCallback(() => {},[])}
-                  findItems={find.bind(null, 3)}
-                  deleteItems={async() => {await Promise.all([])}}
-                  editItem={useCallback(() => {},[])}
-                  tableColumns={tableColumns}
-                  listingLimit={10}
-                  initialPageSize={10}
-                  initialFilter={''}
-                  noItemsFragment={<div></div>}
-                  entityName={i18n.translate('maps.listing.table.entityName', {
-                    defaultMessage: 'map',
-                  })}
-                  entityNamePlural={i18n.translate('maps.listing.table.entityNamePlural', {
-                    defaultMessage: 'maps',
-                  })}
-                  tableListTitle={i18n.translate('maps.listing.table.listTitle', {
-                    defaultMessage: 'Maps',
-                  })}
-                  toastNotifications={notifications.toasts}
-                />
-              </EuiPageContentBody>
-            </EuiPageBody>
-          </EuiPage>
-        </>
+        <div>
+          <Switch>
+            <Route exact path="/create-map" render={(props) => <Map />} />
```

Comment:
should we make constants for these path names?

Reply:
Updated